### DISCUSSION TITLE
blog/posts/2024-01-31-redbrick-maintenance-with-nomad

### DISCUSSION THREAD

**Discussion by giscus** [redbrick/blog#4] [Blog]
# blog/posts/2024-01-31-redbrick-maintenance-with-nomad

https://magma.redbrick.dcu.ie/blog/posts/2024-01-31-redbrick-maintenance-with-nomad.html

**Comment by JedHazaymeh** [comment]
:fire: :fire: :fire:

**Reply by JedHazaymeh** [reply]
### Nice blog!

**Reply by JedHazaymeh** [reply]
*from mobile too~*

**Reply by DistroByte** [reply]
This ain't half bad, nice one!
